commcare_translations.py

```diff
@@ -36,7 +37,17 @@ def load_translations(lang, version=1, commcare_version=None):
         return {}
 
     paths_to_try = []
-    if commcare_version:
+    if commcare_version == 'latest':
+        files = listdir(normpath(join(__file__, "../historical-translations-by-version/")))
```

Comment:
Does this work?  Won't this be `"commcare_translations.py/..historical-translations-by-version/"`
You could do something like `join(dirname(abspath(__file__)), '..historical-translations-by-version/')`